tests/test_http.py

```diff
@@ -284,8 +284,9 @@ def text_response():
     return '<html><h1>Hello, world</h1></html>'
 
 
-def data_response():
-    return {'hello': 'world'}
+def data_response() -> http.Response:
```

Comment:
Let's not change the return type, since that breaks the intent of this test.
Probably need to just leave this one alone at the moment, as I think we need to do more work to properly deal with annotated data response types.